### DISCUSSION TITLE
Canvas: Useful features for the new Canvas panel

### DISCUSSION THREAD

**Discussion by rjulius23** [grafana/grafana#64039] [Dashboards, Panels & Visualizations]
Feature ideas, that would be handy for the canvas:

- Possibility to configure Unit / Layer not for all the elements. Right now it is not possible to mix metrics of different dimensions on the canvas.
- Canvas should be responsive, if I change the canvas panel size it messes it up
- Make it possible to use different connection types like dotted lines, curvy lines, rectanuglar lines (similar to draw.io for example)
- It would be amazing if you can add different Panels from Grafana ( like Pie charts, bar gauges, Gauges)

Thanks,

Gyula

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82837. Feel free to continue the discussion around this there. Thank you!
